feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Search.kt

```diff
@@ -218,14 +225,24 @@ private fun SearchScreen(
 private fun SearchTextField(
     searchWord: String,
     onSearchWordChange: (String) -> Unit,
+    onSearchTextAreaClicked: () -> Unit,
     modifier: Modifier = Modifier,
     onBackClick: () -> Unit = {},
 ) {
     val keyboardController = LocalSoftwareKeyboardController.current
     val focusRequester = remember { FocusRequester() }
+    val interactionSource = remember { MutableInteractionSource() }
+
     LaunchedEffect(Unit) {
         focusRequester.requestFocus()
     }
+    LaunchedEffect(interactionSource) {
+        interactionSource.interactions.collect {
+            if (it is PressInteraction.Release) {
```

Comment:
Can I ask why you use PressInteraction instead of click? I think you have some reason. I'd like to know it for my information.

Reply:
The reason is that we feel it is more natural to judge the completion of the click action when the finger is released.
There is no problem with the press interaction, so it can be replaced 👍